dspace-server-webapp/src/main/java/org/dspace/app/rest/Application.java

```diff
@@ -55,6 +56,7 @@
  */
 @SpringBootApplication
 @EnableScheduling
+@EnableCaching
```

Comment:
I initially enabled a cache for manifests using the spring boot starter cache and ehcache 3. This is mostly a test case for my own benefit.  My sense is that a better strategy is either (1) enable the cache using ehcache2 in a separate config file located in `iiif/cache` or (2) use the current dspace strategy for configuring caches.  Meanwhile, for the spring boot starter cache to work I needed to add the annotation to the application as I have done here.

In general, server-side caching for manifests seems to be reasonable to me and I'm curious if others agree.